src/main/java/sg/ncl/DatasetManager.java

```diff
@@ -1,90 +1,63 @@
 package sg.ncl;
 
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.Map;
-import java.util.Random;
-import java.util.TimeZone;
-
 import sg.ncl.testbed_interface.Dataset;
-import sg.ncl.testbed_interface.Experiment;
+
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class DatasetManager {
 	
-	private final String LICENSE_GPL = "gpl";
-	private final String LICENSE_MIT = "mit";
-	private final String LICENSE_BSD = "bsd";
-	private final String VISIBLITY_PUBLIC = "public";
-	private final String VISIBILITY_PRIVATE = "private";
-	private static DatasetManager DATASET_MANAGER_SINGLETON = null;
 	private HashMap<Integer, Dataset> datasetMap; // datasetID, dataset - holds all the dataset information
 	
-	private DatasetManager() {
-		
+	public DatasetManager() {
 		datasetMap = new HashMap<Integer, Dataset>();
+	}
+
+	public void initSamples() {
+	    String ACCESSIBILITY_OPEN = "open";
+        String ACCESSIBILITY_RESTRICTED = "restricted";
+        String VISIBILITY_PUBLIC = "public";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)